src/tasks/includes/utilities.js

```diff
@@ -13,14 +15,35 @@ const Promise = require('bluebird');
 const utilities = {
 
   /**
-   * Generic error handler for streams called in `watch` tasks (used by gulp-plumber)
+   * Handles the output for any errors that might have been captured
+   * during the build and zip Gulp tasks.
+   *
+   * @memberof slate-cli.utilities
+   */
+  outputErrors: () => {
+    if (!errors.length) {
```

Comment:
Ahh true true... Yeah up to you. If there's already a pattern established in the codebase, it is probably better to stick to that.